sys/net/gnrc/routing/rpl/gnrc_rpl_auto_init.c

```diff
@@ -29,7 +30,10 @@ void auto_init_gnrc_rpl(void)
 {
 #if (GNRC_NETIF_NUMOF == 1)
     gnrc_netif_t *netif = gnrc_netif_iter(NULL);
-    assert(netif != NULL);
+    if (netif == NULL) {
```

Comment:
Looks unrelated ?

Reply:
Fixed and squashed in https://github.com/RIOT-OS/RIOT/pull/10431/commits/acb2380d8231a8358079dffcd2c69399fdaa5d05